protecode/scanning.py

```diff
@@ -568,180 +531,17 @@ def result_matches(resource: cm.Resource, result: pm.AnalysisResult):
 def _retrieve_existing_scan_results(
```

Comment:
Remove leading underscore since this function is not for private use anymore (-> referenced in bdba cli)